src/SFA.DAS.EmployerApprenticeshipsService.Application.UnitTests/Commands/UpdateEnglishFractions/WhenISendTheUpdateEnglishFractionsCommand.cs

```diff
@@ -39,14 +39,14 @@ public void Arrange()
                 new DasEnglishFraction
                 {
                     Id = "1",
-                    DateCalculated = DateTime.Parse(DateTime.Now.AddDays(-20).ToShortDateString()),
+                    DateCalculated = DateTime.Parse(DateTime.Today.AddDays(-20).ToShortDateString()),
                     EmpRef = _employerReference,
                     Amount = 0.45M
                 },
                 new DasEnglishFraction
                 {
                     Id = "2",
-                    DateCalculated = DateTime.Parse(DateTime.Now.AddDays(-10).ToShortDateString()),
+                    DateCalculated = DateTime.Parse(DateTime.Today.AddDays(-10).ToShortDateString()),
```

Comment:
Ditto previous comment

Reply:
Agreed, missed this whilst bulk updating code.